spec/index.html

```diff
@@ -10849,622 +10849,692 @@ <h3>Grammar</h3>
+                <td><code class="gRuleBody"><a href="#rVar">Var</a> | <a href="#riri">iri</a> | <a href="#rRDFLiteral">RDFLiteral</a> | <a href="#rNumericLiteral">NumericLiteral</a> | <a href="#rBooleanLiteral">BooleanLiteral</a> | <a href="#rBlankNode">BlankNode</a> | <a href="#rNIL">NIL</a> | <a href="#rTripleTerm">TripleTerm</a></code></td>
               </tr>
 
               <tr style="vertical-align: baseline">
-                <td><code>[109]&nbsp;&nbsp;</code></td>
-                <td><code><span class="doc-ref" id="rQuotedTriple">QuotedTriple</span></code></td>
+                <td><code>[117]&nbsp;&nbsp;</code></td>
+                <td><code><span class="doc-ref" id="rReifiedTriple">ReifiedTriple</span></code></td>
                 <td>&nbsp;&nbsp;::=&nbsp;&nbsp;</td>
-                <td><code class="gRuleBody"><span class="token">'&lt;&lt;'</span> <a href="#rVarOrTerm">VarOrTerm</a> <a href="#rVerb">Verb</a> <a href="#rVarOrTerm">VarOrTerm</a> <span class="token">'&gt;&gt;'</span></code></td>
+                <td><code class="gRuleBody"><span class="token">'&lt;&lt;'</span> <a href="#rVarOrTerm">VarOrTerm</a> <a href="#rVerb">Verb</a> <a href="#rVarOrTerm">VarOrTerm</a> <a href="#rReifier">Reifier</a>? <span class="token">'&gt;&gt;'</span></code></td>
               </tr>
 
               <tr style="vertical-align: baseline">
-                <td><code>[110]&nbsp;&nbsp;</code></td>
-                <td><code><span class="doc-ref" id="rQuotedTripleData">QuotedTripleData</span></code></td>
+                <td><code>[118]&nbsp;&nbsp;</code></td>
+                <td><code><span class="doc-ref" id="rReifiedTripleData">ReifiedTripleData</span></code></td>
                 <td>&nbsp;&nbsp;::=&nbsp;&nbsp;</td>
-                <td><code class="gRuleBody"><span class="token">'&lt;&lt;'</span> <a href="#rDataValueTerm">DataValueTerm</a> ( <a href="#riri">iri</a> | <span class="token">'a'</span> ) <a href="#rDataValueTerm">DataValueTerm</a> <span class="token">'&gt;&gt;'</span></code></td>
+                <td><code class="gRuleBody"><span class="token">'&lt;&lt;'</span> <a href="#rDataValueTerm">DataValueTerm</a> ( <a href="#riri">iri</a> | <span class="token">'a'</span> ) <a href="#rDataValueTerm">DataValueTerm</a> <a href="#rReifierData">ReifierData</a>? <span class="token">'&gt;&gt;'</span></code></td>
               </tr>
 
               <tr style="vertical-align: baseline">
-                <td><code>[111]&nbsp;&nbsp;</code></td>
+                <td><code>[119]&nbsp;&nbsp;</code></td>
+                <td><code><span class="doc-ref" id="rTripleTerm">TripleTerm</span></code></td>
+                <td>&nbsp;&nbsp;::=&nbsp;&nbsp;</td>
+                <td><code class="gRuleBody">L_TRIPLE <a href="#rVarOrTerm">VarOrTerm</a> <a href="#rVerb">Verb</a> <a href="#rVarOrTerm">VarOrTerm</a> R_TRIPLE</code></td>
```

Comment:
We should make `L_TRIPLE` and `R_TRIPLE` links, as also done for other terminals (e.g., `IRIREF`). That is:
```suggestion
                <td><code class="gRuleBody"><a href="#rL_TRIPLE">L_TRIPLE</a> <a href="#rVarOrTerm">VarOrTerm</a> <a href="#rVerb">Verb</a> <a href="#rVarOrTerm">VarOrTerm</a> <a href="#rR_TRIPLE">R_TRIPLE</a></code></td>
```